### ISSUE TITLE
Update to NDK r25b LTS

### ISSUE THREAD

**Issue by EwoutH** [google/lyra#93]
The Android NDK used currently is a bit outdated, being the r21 LTS version. A new LTS version, r25, has been released, and contains a updated compiler and include updated Android APIs.

It's latest exact version is [`25.1.8937393`](https://developer.android.com/ndk/downloads).

**Comment by mchinen** [comment]
I can't tell if bazel support exists yet (there were some issues raised on the bazel githubs [1](https://github.com/bazelbuild/rules_android/issues/46), [2](https://github.com/bazelbuild/bazel/issues/12889), I see you on some threads too).

I see the new [rules_android_ndk github repo](https://github.com/bazelbuild/rules_android_ndk) exists and claims NDK 25b support, so we might try that.

**Comment by EwoutH** [comment]
Yes, I forgot to mention that. See https://github.com/bazelbuild/bazel/issues/12889#issuecomment-1233565322

**Comment by mchinen** [comment]
Would you be interested in sending us a PR for this @EwoutH ?

**Comment by EwoutH** [comment]
Sorry, I don't think I have the capacity to do that in the near feature, unfortunatly.

**Comment by cpsauer** [comment]
Happened to see this via backlink. Can confirm that the new ndk rules work well as a drop in replacement allowing use of NDK r25, additionally solving a bunch of issues with the old implementation. That said, they don't support building on Windows just yet.
